core/openapi/schemas/extent.yaml

```diff
@@ -1,3 +1,90 @@
+        description: >-
+          One or more bounding boxes that describe the spatial extent of the dataset.
+          In the Core only a single bounding box is supported. Extensions may support
+          additional areas. If multiple areas are provided, the union of the bounding
+          boxes describes the spatial extent.
+        type: array
+        minItems: 1
+        items:
+          description: >-
+            West, south, east, north edges of the bounding box. The coordinates
+            are in the coordinate reference system specified in `crs`. By default
+            this is WGS84 longitude/latitude.
+          type: array
+          minItems: 4
+          maxItems: 6
+          items:
+            type: number
+          example:
+            - -180
+            - -90
+            - 180
+            - 90
+      crs:
+        description: >-
+          Coordinate reference system of the coordinates in the spatial extent
+          (property `bbox`). The default reference system is WGS84 longitude/latitude.
+          In the Core this is the only supported coordinate reference system.
+          Extensions may support additional coordinate reference systems and add
+          additional enum values.
+        type: string
+        enum:
+          - 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+        default: 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+  temporal:
+    description: >-
+      The temporal extent of the features in the collection.
+    type: object
+    properties:
+      bbox:
+        description: >-
+          One or more time intervals that describe the temporal extent of the dataset.
+          In the Core only a single time interval is supported. Extensions may support
+          multiple intervals. If multiple intervals are provided, the union of the
+          intervals describes the temporal extent.
+        type: array
+        minItems: 1
+        items:
+          description: >-
+            Begin and end times of the time interval. The timestamps
+            are in the coordinate reference system specified in `trs`. By default
+            this is the Gregorian calendar.
+          type: array
+          minItems: 2
+          maxItems: 2
+          items:
+            type: string
+            format: date-time
+          example:
+            - '2011-11-11T12:22:11Z'
+            - '2012-11-24T12:32:43Z'
+      trs:
+        description: >-
+          Coordinate reference system of the coordinates in the temporal extent
+          (property `interval`). The default reference system is the Gregorian calendar.
+          In the Core this is the only supported temporal reference system.
+          Extensions may support additional temporal reference systems and add
+          additional enum values.
+        type: string
+        enum:
+          - 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
+        default: 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
+
+TODO
```

Comment:
Is that intended to be in the PR?

Reply:
Oops, of course not, thanks for spotting! Lines 84 and 85 have to be removed.